### DISCUSSION TITLE
User folder with space

### DISCUSSION THREAD

**Discussion by Mr-Vipi** [JanDeDobbeleer/oh-my-posh#960] [Troubleshoot] [answered]
I'm trying to implement one of the themes for my git bash cli in windows, but when I try to implement the script in my .bashrc 
I keep having ```bash: C:/Users/John: No such file or directory``` my user folder is John Doe.
I have tried these codes as below but I'm having always the same issue: 
```
eval "$(oh-my-posh --init --shell bash --config ~/AppData/Local/Programs/oh-my-posh/themes/jandedobbeleer.omp.json)"

eval "$(oh-my-posh --init --shell bash --config 'C/Users/John Doe/AppData/Local/Programs/oh-my-posh/themes/jandedobbeleer.omp.json')"

eval "$(oh-my-posh --init --shell bash --config C/Users/John\ Doe/AppData/Local/Programs/oh-my-posh/themes/jandedobbeleer.omp.json)"

eval "$(oh-my-posh --init --shell bash --config 'C/Users/"John Doe"/AppData/Local/Programs/oh-my-posh/themes/jandedobbeleer.omp.json')"

```
I managed to have no problem with powershell.  Can somebody help me or tell what I miss around?

EDIT: after a month later after updating oh-my-posh I tried again implementing the configuration and magically it works with no issues, I don't know tha cuase of this malfunction previously but now I'm glad it works fine. 
This line made it work
```eval "$(oh-my-posh --init --shell bash --config ~/AppData/Local/Programs/oh-my-posh/themes/jandedobbeleer.omp.json)"```

**Comment by lnu** [comment, marked as answer]
fixed by https://github.com/JanDeDobbeleer/oh-my-posh/commit/f64a8fa51a93939dcc331a4c278332fda389325a
